src/albert/resources/data_templates.py

```diff
@@ -22,15 +30,62 @@ class CSVMapping(BaseAlbertModel):
     )
 
 
-class DataColumnValue(BaseAlbertModel):
-    data_column: DataColumn = Field(exclude=True, default=None)
-    data_column_id: str = Field(alias="id", default=None)
+class AthenaMetadata(BaseAlbertModel):
+    table_name: str | None = Field(default=None, alias="tableName")
+    partition_key: str | None = Field(default=None, alias="partitionKey")
+
+
+class S3KeyReference(BaseAlbertModel):
+    rawfile: str | None = None
+    s3_input: str | None = Field(default=None, alias="s3Input")
+    s3_output: str | None = Field(default=None, alias="s3Output")
```

Comment:
Can these be renamed to something more domain specific? I don't think we should be naming classes based on our AWS service names :)